pkg/controllers/vdb/offlineupgrade_reconciler.go

```diff
@@ -228,7 +230,7 @@ func (o *OfflineUpgradeReconciler) checkNMADeploymentChange(ctx context.Context)
 		return ctrl.Result{}, nil
 	}
 
-	stss, err := o.Finder.FindStatefulSets(ctx, iter.FindExisting)
+	stss, err := o.Finder.FindStatefulSets(ctx, iter.FindExisting, o.SandboxName)
```

Comment:
General comment, could we use the sandbox from the podfacts if its available in the reconciler?